src/commands/scan/cmd-scan-create.mts

```diff
@@ -227,6 +234,9 @@ async function run(
     repo: string
     report?: boolean
   }
+  const { reach } = cli.flags as {
+    reach?: boolean
+  }
```

Comment:
There's a `const` destructuring of flags above the `let` destructuring already. Can you please move this one to be part of that one?